dist/leaflet.css

```diff
@@ -45,7 +45,10 @@
 	}
 /* .leaflet-container svg: reset svg max-width decleration shipped in Joomla! (joomla.org) 3.x */
 /* .leaflet-container img: map is broken in FF if you have max-width: 100% on tiles */
-.leaflet-container .leaflet-overlay-pane svg,
+.leaflet-container .leaflet-overlay-pane svg{

```

Comment:
```suggestion
.leaflet-container .leaflet-overlay-pane svg {

```

Reply:
Fixed, can we add eslint on `leaflet.css` too?

Reply:
@Falke-Design perhaps you can log a separate issue for that?